apps/gutenberg-diff/gutenberg-diff-plugin/blocks/group/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { InnerBlocks, getColorClassName } from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import { registerBlockType } from '@wordpress/blocks';
+
+import './style.scss';
+
+export const name = 'gutenberg-diff/group';
```

Comment:
This should have the same name as the core block, but I just used something different for easier testing alongside the proper Group block on my local setup that's running latest Gutenberg master.